core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/regex/RegexOpDesc.scala

```diff
@@ -41,4 +46,5 @@ class RegexOpDesc extends FilterOpDesc {
       inputPorts = List(InputPort()),
       outputPorts = List(OutputPort())
     )
+
```

Comment:
remove all these changes in this file please

Reply:
done